spec/service_consumers/pact_helper.rb

```diff
@@ -344,13 +347,13 @@ def url_encode(str)
         FactoryGirl.create(:link_change,
                            source_content_id: document_a1.content_id,
                            target_content_id: document_b1.content_id,
-                           action: FactoryGirl.create(:action, user_uid: '11111111-1111-1111-1111-111111111111'),
+                           action: action1,
                            change: 'add')
 
         FactoryGirl.create(:link_change,
                            source_content_id: document_a2.content_id,
                            target_content_id: document_b2.content_id,
-                           action: FactoryGirl.create(:action, user_uid: '22222222-2222-2222-2222-222222222222'),
+                           action: action2,
                            change: 'remove')
```

Comment:
I'm guessing that what @kevindew meant by [1] was make the indentation like [2], where the symbol is on the same line, and the hash keys are two spaces in from `Factory...`.

1: https://github.com/alphagov/publishing-api/pull/1039#discussion_r140194927
2: https://github.com/alphagov/publishing-api/blob/master/spec/service_consumers/pact_helper.rb#L77-L93

Reply:
👍